x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/index/signals_mapping.json

```diff
@@ -133,6 +133,9 @@
                   }
                 }
               },
+              "note": {
+                "type": "text"
+              },
```

Comment:
Looking at the ECS mapping we ingest:
x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/index/ecs_mapping.json

It is using:

```ts
                  "text": {
                    "norms": false,
                    "type": "text"
                  }
```

First time seeing that norm. It looks like it is this:
https://www.elastic.co/guide/en/elasticsearch/reference/current/norms.html

Seeing it also being used for all text fields in auditbeat. Wondering if we want that here.

Reply:
Spoke offline re: ^